tests/sentry/ingest/test_transaction_clusterer.py

```diff
@@ -125,3 +126,40 @@ def test_save_rules():
         rules.update(project, [ReplacementRule("bar"), ReplacementRule("zap")])
     project_rules = rules.get(project)
     assert {"bar": "1334318402", "foo": "1334318401", "zap": "1334318402"}
+
+
+@mock.patch("django.conf.settings.SENTRY_TRANSACTION_CLUSTERER_RUN", True)
+@mock.patch("sentry.ingest.transaction_clusterer.tasks.MERGE_THRESHOLD", 5)
+@pytest.mark.django_db
+def test_run_clusterer_task(default_organization):
+    with Feature({"organizations:transaction-name-clusterer": True}):
+        project1 = Project(id=123, name="project1", organization_id=default_organization.id)
+        project2 = Project(id=223, name="project2", organization_id=default_organization.id)
+        for project in (project1, project2):
+            project.save()
+            for i in range(len(project.name)):
+                _store_transaction_name(project, f"/user/tx-{project.name}-{i}")
+                _store_transaction_name(project, f"/org/tx-{project.name}-{i}")
```

Comment:
I know this is not specifically what we are testing here, but should we put `project.name` in a separate segment so that it's clear that the rules are not shared between projects?

```suggestion
                _store_transaction_name(project, f"/user/tx-{project.name}/{i}")
                _store_transaction_name(project, f"/org/tx-{project.name}/{i}")
```